src/pages/EditReportFieldPage.tsx

```diff
@@ -73,44 +84,78 @@ function EditReportFieldPage({route, policy, report}: EditReportFieldPageProps)
         Navigation.dismissModal(report?.reportID);
     };
 
+    const handleReportFieldDelete = () => {
+        ReportActions.deleteReportField(report.reportID, reportField);
+        Navigation.dismissModal(report?.reportID);
```

Comment:
Before we call Navigation.dismissModal here, we should set isDeleteModalVisible to false to prevent the delay in closing the delete modal  https://github.com/Expensify/App/issues/43496